src/main/java/homework_2/random_chars_table/Main.java

```diff
@@ -0,0 +1,45 @@
+package homework_2.random_chars_table;
+
+import java.util.HashSet;
+
+public class Main {
+    public static void main(String[] args) {
+//        initializeForTests();
+       new RandomCharsTable().run();
+//        isValidLastStringForEven("Even letters - ");
```

Comment:
All excessive comments/commented-out lines of code should be removed. Keep your code clean

Reply:
It seems that i fixed it.